docs/README.md

```diff
@@ -12,4 +12,91 @@ There are no requirements at this time.
+
+“I leave Sisyphus at the foot of the mountain. One always finds one's burden again. But Sisyphus teaches the higher fidelity that negates the gods and raises rocks. He too concludes that all is well. This universe henceforth without a master seems to him neither sterile nor futile. Each atom of that stone, each mineral flake of that night-filled mountain, in itself, forms a world. The struggle itself toward the heights is enough to fill a man's heart. One must imagine Sisyphus happy.”
+
+― Albert Camus
+
+SUMMARY:
+
+Pac-man is a 2-dimensional game in which a sentient yellow circle with a mouth compulsively eats smaller yellow circles, whilst navigating through an endless maze.  As Pac-man navigates the passages of his maze, eating “pellets”, he is haunted by the spectre of four colored ghosts who wish to end his Sisyphean existence by killing him through their touch.  In each maze, there are four flashing pellets of a larger size than regular pellets.  These “power pellets” give Pac-man invulnerability to the ghost touch attack, and at lower levels he gains the ability to consume his spectral pursuers for a limited time.  Pac-man gains points for destroying the ghosts while in power pellet mode, visually cued by a change in ghost color to a unanimous deep blue.  He also gains points for eating regular pellets, power pellets, and fruits.  After earning ten thousand points, Pac-man can gain an additional life until he reaches a maximum of five concurrent lives. Once all pellets on a level are consumed, the screen will flash four times and the maze resets.  When the maze resets, Pac-man returns to his starting position, the ghosts return to their pin, and the pellets repopulate the maze.  As levels progress, the maze layout changes and the red ghost will move faster.  The final level is 256.  This is the “kill level” because the game runs out of memory.  On level 256, the game loads half of a normal map with the right half corrupted.  The final level is impossible to complete. A cruel joke or an allegory for the impossibility of nirvana?
+
+Who is Pac-man? Why does he eat pellets? Why do the four ghosts pursue him?  Does Pac-man destroy his own universe at level 256, only to see it reborn anew?  It is possible that Pac-man himself does not know, but we can attempt to gain personal understanding through observation of his ceaseless performance of destruction and rebirth.
+
+
+PAC-MAN (the man)
+
+Pac-man is yellow and circular. His mouth opens and closes continuously.  He initially moves slightly faster than the ghosts.  He begins each level moving to the left.  He will move forward until he hits a wall.  When he hits a wall, he will face the wall and his mouth will continue to open and close, but he will not move in any direction.  Even when not moving, Pac-man will be pursued by the ghosts.  He must continue to move and to eat, or he will perish and his torment will start anew. 
```

Comment:
How fast is 'slightly faster' than the ghosts?

Reply:
that is a great question that I could not find a number for.  But, I had to stop going down the rabbit hole.